client/packages/core/src/Reactor.js

```diff
@@ -1509,18 +1509,31 @@ export default class Reactor {
     return res;
   }
 
-  async signOut() {
-    const currentUser = await this.getCurrentUser();
+  potentiallyInvalidateToken(currentUser, opts) {
     const refreshToken = currentUser?.user?.refresh_token;
-    if (refreshToken) {
-      try {
-        await authAPI.signOut({
-          apiURI: this.config.apiURI,
-          appId: this.config.appId,
-          refreshToken,
-        });
-      } catch (e) {}
+    if (!refreshToken) {
+      return;
+    }
+    const wantsToSkip = opts.invalidateToken === false;
+    if (wantsToSkip) {
+      log.info('[auth-invalidate] skipped invalidateToken');
+      return;
     }
+    authAPI
```

Comment:
Removed the await here -- my reasoning was that we may not want this to block the signout speed

Reply:
I think we should block here because the request might fail, in which case you should try again.

Reply:
(Discussed offline. We can't quite do this, as it would block offline sign out)